templates/es2015/server.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Module dependencies.
+ */
+
+import app from './app';
+import {default as debugModule} from 'debug';
```

Comment:
Also, the default module has no correlation with the import name - you could call it whatever you want, including my comment above.